### ISSUE TITLE
Localization MSF module dataset download link

### ISSUE THREAD

**Issue by mengen100** [ApolloAuto/apollo#13576]
Hi 
I suppose I am using the old link (http://data.apollo.auto/help?name=sensor%20data&data_key=multisensor&data_type=1&locale=en-us&lang=en) to download the dataset, once I click on **Download**, it shows me this bucket error.

![ss1](https://user-images.githubusercontent.com/62421236/111421597-a7fc3300-86c3-11eb-815b-0d01b28a2b91.png)

Is there anyone having the same issue? Anyone knows where to download the dataset?
Or anyone could _share_ the dataset you've downloaded? .
I really need it in order to run the MSF localization module.. Any help is appreciated, thanks!!

Btw I'm following this instruction (https://www.fzb.me/apollo/howto/how_to_run_MSF_localization_module_on_your_local_computer.html)

**Comment by Juntu-hub** [comment]
I have the same problem, have you solved it?

**Comment by mengen100** [comment]
> I have the same problem, have you solved it?

@Juntu-hub Nope, probably because either not many people are using MSF Localization module or that documentation is obsolete..

**Comment by mengen100** [comment]
@Juntu-hub I found another link: https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_generate_local_map_for_MSF_localization_module.md which seems a newer version of the instruction. We are required send a email request with email address, affliction and application to request for the data. Please find more details from the above link. 
@panfengsu Hi admin, I think "How to run MSF Localization module" documentation (https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_run_MSF_localization_module_on_your_local_computer.md) should be updated in adherence with "How to Generate Local Map" (https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_generate_local_map_for_MSF_localization_module.md).

**Comment by Juntu-hub** [comment]
@mengen100 Thank you,  I have sent a email and got the data.

**Comment by AlexandrZabolotny** [comment]
> @mengen100 Thank you, I have sent a email and got the data.

can You send me this data too?

**Comment by jay80253504** [comment]
@Juntu-hub hello，Could you sent me the data please？ My email is 745874655@qq.com.  Thank you very much!

**Comment by blacksheep0303** [comment]
@Juntu-hub hello,could you sent me the data please?My email is fyqbuaa@163.com.Thanks a lot !!!

**Comment by jxhstrive** [comment]
@Juntu-hub Hi, could you pls send me the data? My email is jiangxuhui2582@126.com, thanks a lot!

**Comment by jxhstrive** [comment]
@Juntu-hub Hi, could you pls send me the data? My email is jiangxuhui2582@126.com, thanks a lot!

**Comment by lucianzhong** [comment]
@Juntu-hub, could you please send me the data? My email is 373605178@qq.com, thanks a lot

**Comment by lidarTJHHP** [comment]
@Juntu-hub Hi, could you pls send me the data? My email is 767316704@qq.com, thanks a lot!
